src/components/list/moto_list.tsx

```diff
@@ -33,25 +63,33 @@ export default function MotorcycleList({ motorcycles, setMotorcycles }: Motorcyc
   };
 
   const handleSaveEdit = () => {
-    if (editMotorcycle) {
+    if (editMotorcycle && !hasErrors) {
       const updatedMotorcycles = [...motorcycles];
```

Comment:
You don't need to check if there is error in handleSave, since this function should not be triggered if there is errors.
See the next comment